migration/src/m0000010_init.rs

```diff
@@ -15,12 +15,18 @@ impl MigrationTrait for Migration {
         // Delete the SET statements at the start of the file
         // Move the drop statements at the start to m0000010_init_down.sql
 
-        let x = manager
+        manager
+            .get_connection()
+            .execute_unprepared(include_str!("m0000005_admin_up.sql"))
+            .await
+            .map(|_| ())?;
+
```

Comment:
How is it that this executes 'admin level' commands, but it doesn't in the next statement for m0000010_init_up.sql?

Reply:
@chirino, 

That's a good point. I was focused on running that part as "manager" but didn't realize that the other migrations are running at the same level !

The SeaOrm's manager has only the privileges of the application user's roles allocated to it. Therefore that won't work and the extension creation must be removed from the migration.